### PR TITLE
Enpoint to get the list of categories for a given list of binary protected values

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

**Review by erikasv — APPROVED** [review]

File: services/backend/src/routes/protected_areas.js [hunk 1]
```diff
@@ -26,6 +26,31 @@ module.exports = (errorHandler, paService) => {
...
   )));
 
+  /**
+   * @apiGroup geofence_pa
+   * @api {get} /pa/categories/binary_protected/:binary_protected listCategoriesByBinaryProtected
+   * @apiName listCategoriesByBinaryProtected
+   * @apiVersion 0.1.0
+   * @apiDescription
+   * List available protected area categories for the given binary protected values
+   *
```
Comment (erikasv) [inline services/backend/src/routes/protected_areas.js:36]: Sorry, forgot to mention, missing param documentation

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/backend/src/routes/protected_areas.js (hunk 1 above)
Comment (erikasv) [reply services/backend/src/routes/protected_areas.js:36]: and please specify that is a list separated by `&`

Comment (cazapatamar) [reply services/backend/src/routes/protected_areas.js:36]: Done

Comment (cazapatamar) [reply services/backend/src/routes/protected_areas.js:36]: Done

**Review by erikasv — APPROVED** [review]

**Pull request merged**